stable/kong/values.yaml

```diff
@@ -14,6 +14,7 @@ admin:
   #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 
   # HTTPS traffic on the admin port
+  # if set to false also set readinessProbe and livenessProbe httpGet scheme's to 'HTTP'
```

Comment:
Sorry, should have recommended it before,

`# if set to false, you would also need to set both readinessProbe and livenessProbe httpGet scheme to 'HTTP'`